content/user/upgrading/convert_to_utf8.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Technical - Converting from iso-8859-1 to utf8
-description: Zen Cart Converting from iso-8859-1 to utf8
+description: Latin1 has got to go
```

Comment:
?

Reply:
LOL.  I am open to any description that does not simply restate the title.